octobot_trading/orders/order_state.py

```diff
@@ -0,0 +1,78 @@
+#  Drakkar-Software OctoBot-Trading
+#  Copyright (c) Drakkar-Software, All rights reserved.
+#
+#  This library is free software; you can redistribute it and/or
+#  modify it under the terms of the GNU Lesser General Public
+#  License as published by the Free Software Foundation; either
+#  version 3.0 of the License, or (at your option) any later version.
+#
+#  This library is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+#  Lesser General Public License for more details.
+#
+#  You should have received a copy of the GNU Lesser General Public
+#  License along with this library.
+from octobot_trading.enums import OrderStates
+from octobot_trading.util.initializable import Initializable
+
+
+class OrderState(Initializable):
+    def __init__(self, order, from_exchange_data):
+        super().__init__()
+
+        # related order
+        self.order = order
+
+        # default state
+        self.state = OrderStates.UNKNOWN
+
+        # if this state has been created from exchange data or OctoBot internal mechanism
+        self.from_exchange_data = from_exchange_data
```

Comment:
we should name this param `is_from_exchange_data ` or something like this to underline the fact that it's a bool